### PR TITLE
Add a blurb about port allocation

### PR DESCRIPTION
**Pull request by pshafton**
This is a first pass at the port allocation faq entry. Let me know if you want me to add more detail here.

### CODE REVIEW

**Review by shubcodes — APPROVED** [review]
This is the correct response to the customer query about open ports.

File: docs/faq/faq.md [hunk 1]
```diff
@@ -79,3 +79,12 @@ _en-grok_
 
 The first prototype for ngrok was committed on [March 20th,
 2013](https://github.com/inconshreveable/ngrok/commit/8f4795ecac7f92c6b5a8c8970c65f26e5315fe4e).
+
+## Why does port scanning my ngrok domain show other open ports {#open-ports-on-domain}
+
+ngrok uses a shared set of servers to service both HTTP(s)/TLS and TCP traffic. 
+All HTTP(s)/TLS traffic is serviced on ports 80 and 443. For TCP traffic each
```
Comment (krwenholz) [inline docs/faq/faq.md:86]: Is it worth pointing out that SNI is what gets traffic routed correctly on those ports? If I recall correctly, we had a similar conversation with our pentesters a few years back to get language clarified in our first report.

**Review by krwenholz — APPROVED** [review]
Awesome. Good level of detail. Thanks, Peter.

File: docs/faq/faq.md (hunk 1 above)
Comment (pshafton) [reply docs/faq/faq.md]: I added a section about SNI and protecting traffic.

**Pull request merged**